gptme/llm/llm_anthropic.py

```diff
@@ -13,6 +15,20 @@
 anthropic: "Anthropic | None" = None
 
 
+class AnthropicModelMeta(ModelMeta):
+    supports_file = True
+
+    def prepare_file(self, media_type, data):
+        return {
+            "type": "image",
+            "source": {
+                "type": "base64",
+                "media_type": media_type,
+                "data": data,
+            },
+        }
```

Comment:
Why inherit by provider from ModelMeta? Use a dataclass `field`. 
Not every model from the same provider have the same capabilities (CLaude 3.5 Haiku doesn't have vision support, for example).

I think `supports_vision` is a better name.

I don't think it should be the job of `ModelMeta` to prepare the file, it should be the job of the provider implementation when calling `msgs2dicts`.

Reply:
Removed the class inheritance and using a plain property.

I removed the`supports_vision` property as we don't need it anymore after the file refactoring.